test/Dockerfile.ocimanifest

```diff
@@ -0,0 +1,18 @@
+# This Dockerfile is used to test image with an OCI manifest (issue #1184). It
+# must be built with Podman; version 3.0.1 works.
```

Comment:
Do we have version protection for this test? What happens when a non-`3.0.1` version is tested?